crates/swc_ecma_minifier/tests/fixture/next/feedback-util-promisify/chunks/pages/_app-72ad41192608e93a/output.js

```diff
@@ -131,7 +131,7 @@
                             return t2[e >> 18 & 63] + t2[e >> 12 & 63] + t2[e >> 6 & 63] + t2[63 & e];
                         }
                         function encodeChunk(e, r, t) {
-                            for(var f, n = [], i = r; i < t; i += 3)f = (e[i] << 16 & 16711680) + (e[i + 1] << 8 & 65280) + (255 & e[i + 2]), n.push(tripletToBase64(f));
+                            for(var n = [], i = r; i < t; i += 3)n.push(tripletToBase64((e[i] << 16 & 16711680) + (e[i + 1] << 8 & 65280) + (255 & e[i + 2])));
```

Comment:
`f` is inlined into `n.push(tripletToBase64(f));` because `n.push`  (property accesss) nor `tripletToBase64` (ident ref) does not modify `f` or the initializer of `f`